Python.md

```diff
@@ -40,7 +85,37 @@ result = connection.execute(sqlalchemy.text("""
            show_gravatar
       FROM "user"
      WHERE musicbrainz_id IN :musicbrainz_usernames
-"""), {
-    'musicbrainz_usernames': tuple(usernames),
-})
+""")
+result = connection.execute(query, {'musicbrainz_usernames': tuple(usernames)})
 ```
+
+It is preferred to assign the query to a separate variable and not include it inside the
+`execute` call.
```

Comment:
Not to say that I think all variables that are used once should be removed. Sometimes it's useful to split out logical parts to make code easier to understand. But I don't think that's the case with a query string.